lib/delayed/job_groups/complete_stuck_job_groups_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Delayed
+  module JobGroups
+    class CompleteStuckJobGroupsJob
+      if defined?(Delayed::Extensions::SystemJobWithoutSecurityContext)
+        include Delayed::Extensions::SystemJobWithoutSecurityContext
+      end
+
+      class << self
+        def enqueue(**kwargs)
+          Delayed::Job.enqueue(new, **kwargs)
+        end
+      end
+
+      def perform
+        Delayed::JobGroups::JobGroup.ready.find_each(&:check_for_completion)
```

Comment:
Yes! I didn't do this because I was rushing and didn't want to figure out how `having` works again 😂 But yes, I updated this. I also updated the `check_for_completion` to add a flag to skip the active job check, since we don't need to check that twice.